def generate_errors():
    return None


def generate_error_single():
    return None
